docs/source/Grammar.md

```diff
@@ -10,7 +10,7 @@ include = `include` string\_constant `;`
 
 namespace\_decl = `namespace` ident ( `.` ident )* `;`
 
-attribute\_decl = `attribute` ident | `"`ident`"` `;`
+attribute\_decl = `attribute` ident | `"` ident `"` `;`
```

Comment:
There is some weird formatting in this line in the generated documentation page. I think adding the whitespace should fix it.